eng/InstallRuntimes.proj

```diff
@@ -22,8 +24,11 @@
     <BuildArch Condition="'$(BuildArch)' == ''">$(Platform)</BuildArch>
     <BuildArch Condition="'$(BuildArch)' == ''">x64</BuildArch>
     <CommonInstallArgs>-architecture $(BuildArch)</CommonInstallArgs>
-    <TestConfigFileName Condition="'$(BuildArch)' != 'x86'">$(DotNetRoot)Debugger.Tests.Versions.txt</TestConfigFileName>
-    <TestConfigFileName Condition="'$(BuildArch)' == 'x86'">$(DotNetRoot)x86\Debugger.Tests.Versions.txt</TestConfigFileName>
+    <DotNetInstallRoot Condition="'$(BuildArch)' != 'x86'">$(DotNetRoot)</DotNetInstallRoot>
+    <DotNetInstallRoot Condition="'$(BuildArch)' == 'x86'">$(DotNetRoot)x86\</DotNetInstallRoot>
+    <DotNetInstallDir>$(DotNetInstallRoot.Replace("\", "\\"))shared\\Microsoft.NETCore.App\\$(MicrosoftNETCoreAppVersion)\\</DotNetInstallDir>
```

Comment:
Why not use forward slashes that MSBuild understands natively? Or even better, I believe this works:
```suggestion
<DotNetInstallRoot>$([MSBuild]::NormalizeDirectory('$(DotNetRoot)')</DotNetInstallRoot>
<DotNetInstallRoot Condition="'$(BuildArch)' == 'x86'">$([MSBuild]::NormalizeDirectory('$(DotNetRoot)', 'x86'))</DotNetInstallRoot>
<DotNetInstallDir>$([MSBuild]::NormalizeDirectory('$(DotNetRoot)', 'shared' 'Microsoft.NETCore.App', '$(MicrosoftNETCoreAppVersion)'))</DotNetInstallDir>
```

Reply:
"DotNetInstallDir" needs to have the "\\" (double-backslashes) explicit because they need to be in the string put in the reg file. I assume normalizing this property would remove the \\.